app/helper/jwtDecode.js

```diff
@@ -1,3 +1,3 @@
-const jwt = require('jwt-simple');
+const jwt = require('jsonwebtoken');
 
-module.exports = async (...args) => jwt.decode(...args);
+module.exports = async (...args) => jwt.verify(...args);
```

Comment:
why async?

Reply:
I made it like that to be able to use "await". But this is old, in this branch I only change the function from "decode" to "verify"